jekyll/_cci2/server/installation/phase-1-prerequisites.adoc

```diff
@@ -369,6 +369,21 @@ gcloud projects add-iam-policy-binding <PROJECT_ID> \
 certbot certonly --dns-google --dns-google-credentials <KEY_FILE> -d "<CIRCLECI_SERVER_DOMAIN>" -d "app.<CIRCLECI_SERVER_DOMAIN>"
 ----
 
+[#aws-certmanager]
+=== AWS Certificate Manager
+Instead of provisioning your own TLS certificates, if you're setting up circleci-server in an AWS environment, you can have AWS provision TLS certificates using Certificate Manager.
```

Comment:
```suggestion
Instead of provisioning your own TLS certificates, if you are setting up CircleCI Server in an AWS environment, you can have AWS provision TLS certificates using Certificate Manager.
```